libs/newsletter-workflow/src/lib/steps/launchNewsletter/doLaunchLayout.ts

```diff
@@ -0,0 +1,44 @@
+import type { LaunchService } from '@newsletters-nx/newsletters-data-client';
+import type { WizardStepLayout } from '@newsletters-nx/state-machine';
+import { executeLaunch } from '../../executeLaunch';
+import { getStringValuesFromRecord } from '../../getValuesFromRecord';
+import { regExPatterns } from '../../regExPatterns';
+
+// TODO - this page is a placeholder until we implement automation
+// I have added it so that both Ophan and Edit Ophan point to a single place from which to call the Launch functionality
+const markdownTemplate = `
```

Comment:
is this comment still current? I can remember if I copied it from when this was part of the main Draft Wizard.

Reply:
Yes, I added that comment as part of this PR.  I don't expect this page to stay as it is, but it seemed better than having a launch button on both the Ophan and Edit Ophan steps, given that additional pages will inevitably be inserted between those and the end of the wizard (either in the mvp or afterwards)